### ISSUE TITLE
有关代码的问题

### ISSUE THREAD

**Issue by WestHan33** [SunYM2020/UA-CMDet#15]
尊敬的作者您好，感谢您的工作对我帮助很大。在阅读代码时我有一个困惑：在detectors/UACMDet.py代码中，为什么rgb+ir模型在进行concat操作前要使用.detach()操作防止梯度更新呢？我的理解是，双模态分支应该进行学习才能更好地融合。谢谢作者，期待您的答复。
        for i in range(len(x_r)):
            concat = torch.cat((x_r[i].detach(), x_i[i].detach()), dim=1)          
            concat = self.fusion_conv(concat)
            x_f.append(concat)
        x_f = tuple(x_f)
